var/spack/repos/builtin/packages/slepc/package.py

```diff
@@ -12,7 +12,7 @@ class Slepc(Package):
     """Scalable Library for Eigenvalue Problem Computations."""
 
     homepage = "http://slepc.upv.es"
```

Comment:
Can you update the homepage to https while you're at it?

Reply:
Done